eng/common/testproxy/test-proxy-tool.yml

```diff
@@ -0,0 +1,52 @@
+parameters:
+  rootFolder: '$(Build.SourcesDirectory)'
+
+steps:
+  - pwsh: |
+        . $(Build.SourcesDirectory)/eng/common/scripts/common.ps1
+
+        if ($FindArtifactForApiReviewFn -and (Test-Path "Function:$FindArtifactForApiReviewFn"))
+        {
+            &$FindArtifactForApiReviewFn
```

Comment:
How ds this script have anything to do with loading the certificate?

Reply:
I didn't update this appropriately from my test source :| 

Thanks for the catch 👍